### DISCUSSION TITLE
PRF-values at entity-level for NER model

### DISCUSSION THREAD

**Discussion by CarolLuca** [explosion/spaCy#8530] [Help: Coding & Implementations] [answered]
Hello!

I have a question regarding the evaluation of the model with the help of the function `scorer`: are the performance results at entity- or token-level? 

If the latter, I would be grateful to be proffered some hints about how can I find the prf-values at entity-level.

Also, if the context is helpful, here is how I implemented an evaluation function: 
[https://github.com/explosion/spaCy/discussions/8178].

Thanks in advance!

**Comment by adrianeboyd** [comment, marked as answer]
Hi, here is a thread with more information about the NER scoring method: #7494. It's micro-PRF at the entity level.

**Reply by CarolLuca** [reply]
Thank you for the clarification!
